net/stubby/files/stubby.yml

```diff
@@ -24,6 +24,14 @@ upstream_recursive_servers:
 # # Cloudflare IPv6
   - address_data: 2606:4700:4700::1111
     tls_auth_name: "cloudflare-dns.com"
+    tls_pubkey_pinset:
```

Comment:
From what I have read, it is not currently recommended to do this with the cloudflare service.

https://dnsprivacy.org/wiki/display/DP/DNS+Privacy+Public+Resolvers